Changes

```diff
@@ -328,6 +328,9 @@ Features wishes:
   (Florian Angeletti)
 - GPR#252: improve build instructions in MSVC Windows README
   (Philip Daian)
+* GPR#170: Parse arbitrary precision integers and accept a single [A-Za-z]
```

Comment:
Could you maybe add a line to clarify why this can break existing programs? It would probably suffice to just show an example: `let a = 1 in (+) 123a` was accepted but is now rejected. This would reassure users that, unless they did horrendous stuff, they should be quite safe.

(You should feel free to squash your Changelog commits together.)